app/backend/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-bullseye
```

Comment:
Should we use 3.11 to match the devcontainer? Or switch both to 3.12?

Reply:
I'll change both to 3.12, given that 3.13 is now out (so 3.12 is considered fairly stable).